app/src/examples/index.ts

```diff
@@ -312,6 +313,11 @@ export const EXAMPLES: Record<string, Example> = {
     title: 'Volume slider & sensor',
     screen: VolumeExample,
   },
+  TransitionProgressExample: {
+    icon: '☕',
+    title: 'Progress transition example',
+    screen: ProgressTransitionExample,
```

Comment:
`TransitionProgress` or `ProgressTransition`?

Reply:
There's no consistency on that matter in the codebase

![image](https://github.com/software-mansion/react-native-reanimated/assets/39658211/ef472885-e6bd-4ccc-aba6-209bcce7ff94)
![image](https://github.com/software-mansion/react-native-reanimated/assets/39658211/1c4fd53e-afa8-4d7e-9a1a-0cad3a87c3d7)

@piaskowyk to which naming should we stick to?

Reply:
`TransitionProgressEvent` - this is originally event name comes from `rn-screens`

`ProgressTransition` - this is shared element transition bases on progres value that comes from screen's `TransitionProgressEvent`

In summary we should stick with `ProgressTransition` I think.

Reply:
Alright, I'll try to keep the "progress transition" name consistent when creating docs for this in the future as well

Fixed in 9b42b9e